lib/components/user/places/place-editor.js

```diff
@@ -75,11 +78,11 @@ class PlaceEditor extends Component {
 
           <FormGroup>
             <ToggleButtonGroup
+              defaultValue={place.type}
```

Comment:
A couple of styling tweaks. Could you:
- add `bsSize='small'` to ToggleButton?
- change LargeIcon font-size to 125%
- change ToggleButton style to `padding: '5px'` (i.e., change sides from 10px to 5px).

Reply:
I propose something in between. Your suggestion is a little small.
![image](https://user-images.githubusercontent.com/56846598/107092459-a476b180-67d1-11eb-89b2-e3b1dc3173ee.png)

Reply:
See the proposed tradeoff in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/436b93da0ef77b9c75628f92d29cf8313085d163.